### ISSUE TITLE
The master branch build fails to start up

### ISSUE THREAD

**Issue by ashinpan** [texstudio-org/texstudio#2844]
### Environment
- TeXstudio:  master branch
- Qt: 6.4.2
- OS: Ubuntu 22.10 Cinnamon
- TeX distribution: texlive

### Expected behavior

The texstudio should normally start. 

### Actual behavior

When texstudio was started after successful compilation and installation,  it failed to start up, and gave the following message:

"texstudio: error while loading shared libraries: libQt6PrintSupport.so.6: cannot open shared object file: No such file or directory"

Actually, I have the supposedly "missing" file in my Qt folder. I don't know why texstudio failed to find it, even though it had been built with the same Qt distro. 

### How to reproduce

Use cmake-gui to build the master branch. Accept all defaults but the INSTALL_PREFIX can be changed if one wishes.  

The build and install should be successful, but it would actually fail to run.

**Comment by sunderme** [comment]
development is done on ubuntu against qt 6.4.2, obviously with no crashes.

So, it looks like a build issue. I would strongly recommend to use qt-cmake as generator command.

**Comment by ashinpan** [comment]
@sunderme 

I have fixed the issue. I would like to elaborate on how I went wrong so that others can avoid my mistakes. 

With the previous builds, qt5 was the default, so I just used qt 5.15.6 (ubuntu) to build texstudio and install it in the home folder. No problem at the time.

Then, I come to need qt6 for building some other software, so I installed qt6 in the home folder. To make it work, I had to define qt6.conf in /etc/ld.so.conf.d/. I also had to change the user path in the .profile (home) so that qmake (qt6) in the home folder can be discovered by the system. At the time, qt6 version is 6.4.1. 

Later, I upgraded to qt 6.4.2 and removed 6.4.1. Accordingly, I also modified the user path in .profile (home) for the new qmake. But, I forgot to update qt6.conf in  /etc/ld.so.conf.d/.

Now, cmake is mandatory. Due to the reasons that I don't understand, texstudio can find qt6 during the build time, but fails to do so during the runtime.  Only when I remember to check and update qt6.conf that texstudio can run properly again.

From this issue, we can derive two items as food for thought:

(1) Qt6 in my machine is installed in the home folder to use in building software that would also be installed in the home folder. In this case, why should I need to define qt6.conf in the system folder? But Ubuntu insists on this. If Ubuntu were, as before, to have allowed me to define the library path of Qt6 in .profile (home), I would not have made this mistake. 

(2) If texstudio can find qt6, or anything for this matter, during the build time, why should it fail to do so during the runtime? I honestly do not understand it.
